### PR TITLE
Run codespell on code

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: pkg/chunked/compression.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func readEstargzChunkedManifest(blobStream ImageSourceSeekable, blobSize int64,
 	   - 2  bytes  Extra: SI1 = 'S', SI2 = 'G'
 	   - 2  bytes  Extra: LEN = 22 (16 hex digits + len("STARGZ"))
 	   - 22 bytes  Extra: subfield = fmt.Sprintf("%016xSTARGZ", offsetOfTOC)
-	   - 5  bytes  flate header: BFINAL = 1(last block), BTYPE = 0(non-compressed block), LEN = 0
+	   - 5  bytes  flat header: BFINAL = 1(last block), BTYPE = 0(non-compressed block), LEN = 0
```
Comment (mtrmac) [inline pkg/chunked/compression.go:90]: https://pkg.go.dev/compress/flate , this was correct.

**Pull request merged**